BP/history/characters/khitan.txt

```diff
@@ -12,1491 +12,39 @@
 	}
 }
 
-247343 = {
```

Comment:
Put in _lt_rascek_changed_characters.txt; the holders of d_tuyuhun were listed as khitan tengrists; Khitans however, seem to be more asociated with Manchuria (see <https://upload.wikimedia.org/wikipedia/commons/0/0f/East-Hem_565ad.jpg> for example) rather than that far west. So all the holders of d_tuyuhun were converted to the Tuyuhun culture instead; it was an extant language for the the Tuyuhuns during the period.

Reply:
Other Khitan Chars were moved to be Xianbei Dharmaguptaka instead of Khitan Buddhists as the Tuoba (which should probably be a separate culture under CK3's culture system [because of their settlement/rule in China]) which is a more accurate model for those characters.